Resources/crontab.yml

```diff
@@ -88,3 +88,28 @@ real:
         command: 'bin/console opendata -c'
         type: php
         nice: true
+    -
+        schedule: '0 0 1 10 *'
+        command: 'bin/console certificate_reminder >/dev/null 2>&1'
+        type: php
+        nice: true
+    -
+        schedule: '0 0 1 12 *'
+        command: 'bin/console certificate_reminder >/dev/null 2>&1'
+        type: php
+        nice: true
+    -
+        schedule: '0 0 15 12'
```

Comment:
Is it possible that an asterisk is missing here? I remember that there are 5 parameters when setting a cron task.